geode-core/src/main/java/org/apache/geode/distributed/internal/DistributionAdvisor.java

```diff
@@ -172,6 +174,10 @@
    */
   private long currentVersionOpCount;
 
+  private Map<Thread, ExceptionWrapper> currentVersionOperationThreads = new HashMap<>();
```

Comment:
should these be final?

Reply:
I'll make them final and set them to Collections.emptyMap() if debug logging isn't enabled.